pkg/controller/direct/bigqueryconnection/connection_controller.go

```diff
@@ -157,19 +118,19 @@ var _ directbase.Adapter = &Adapter{}
 func (a *Adapter) Find(ctx context.Context) (bool, error) {
 	log := klog.FromContext(ctx).WithName(ctrlName)
 
-	log.V(2).Info("getting BigQueryConnectionConnection", "name", a.id.AsExternalRef())
+	log.V(2).Info("getting BigQueryConnectionConnection", "name", a.desired.Name)
```

Comment:
Probably `a.id.External`

Reply:
a.id.External could be empty because this is a service-generated resource we store the serviceID in external.